### PR TITLE
ROX-7026: Re-enable ChaosMonkey in AdmissionControllerTest.

### PR DESCRIPTION
**Pull request by porridge**
## Description

This PR contains two kinds of changes:
- the first one enables `ChaosMonkey` in `AdmissionControllerTest`
- the remaining ones are temporary changes for debugging, *not meant to be merged*

## Checklist
- [ ] Investigated and inspected CI test results
- [x] Unit test and regression tests added
- [x] ~Evaluated and added CHANGELOG entry if required~
- [x] ~Determined and documented upgrade steps~
- [x] ~Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))~

## Testing Performed

Depending on CI.

### CODE REVIEW

**Comment by porridge** [comment]
Looks like there are plenty of failures in the test now, so you can refrain from reviewing for now while I investigate, @gavin-stackrox :-)

File: qa-tests-backend/Makefile [hunk 1]
```diff
@@ -57,6 +57,60 @@ bat-test: compile
 	@echo "+ $@"
 	$(GRADLE) testBegin testBAT $(GRADLE_TEST_ARGS)
 
+.PHONY: act-test
+act-test: compile
+	@echo "+ $@"
+	$(GRADLE) test --tests=AdmissionControllerTest $(GRADLE_TEST_ARGS)
```
Comment (clickboo) [inline qa-tests-backend/Makefile:63]: Line 62-112 are identical?  🤔

**Pull request closed without merging**